app/src/lib/git/cherry-pick.ts

```diff
@@ -41,6 +49,74 @@ export enum CherryPickResult {
   Error = 'Error',
 }
 
+/**
+ * A parser to read and emit cherry pick progress from Git `stdout`.
+ *
+ * Each successful cherry picked commit outputs a set of lines similar to the
+ * following example:
+ *    [branchName commitSha] commitSummary
+ *    Date: timestamp
+ *    1 file changed, 1 insertion(+)
+ *    create mode 100644 filename
```

Comment:
Kind of minor, but since this is a parser and format is important, I think this commend should reflect the right indentation of the output:
```suggestion
 *    [branchName commitSha] commitSummary
 *      Date: timestamp
 *      1 file changed, 1 insertion(+)
 *      create mode 100644 filename
```